### PR TITLE
updating documentation

### PR DESCRIPTION
**Pull request by brycahta**
*Issue #, if available:* N/A

*Description of changes:*
* Updating documentation to align with these changes: https://github.com/awslabs/amazon-ec2-instance-qualifier/pull/10 ; therefore, this PR should be reviewed+merged after

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by brycahta** [comment]
should add a _defaults table_ similar to AEMM

--> i'll do this after CX improvements as some of the flags and values will change

File: README.md [hunk 1]
```diff
@@ -45,11 +47,12 @@ The instance qualifier is an open source command line tool that automates benchm
...
-  * **An IAM Role**: attached with AmazonS3FullAccess policy to allow instances to access the bucket without AWS credential configuration
+  * **An IAM Role**: attached with AmazonS3FullAccess and CloudWatchAgentServerPolicy policies to allow instances to access the bucket and emit CloudWatch metrics, respectively
   * **Launch Templates**: used to launch auto scaling group and instances
   * **An Auto Scaling Group**: the reason we use auto scaling group to manage all instances is that an one-time action can be scheduled to terminate all instances in the group after timeout to ensure the user is not excessively charged
   * **EC2 Instances**
-* An S3 bucket is also created and won't be deleted after the run, which contains the raw data if deep dive is needed
+* An **S3 bucket** containing the raw data of an Instance-Qualifier run is also created; however, this artifact is persisted by default
```
Comment (haugenj) [inline README.md:54]: The nittiest of nits, but the bolding of 'S3 bucket' should include the 'An' to match the other lines. Or probably better the bold should be removed from the 'A' and 'An's on the other lines

**Review by haugenj — APPROVED** [review]
👍 nice

File: README.md (hunk 1 above)
Comment (brycahta) [reply README.md:54]: 👍🏼  I like attention to detail! will update in next revision; holding off on this merge as I need the other PR to be reviewed/merged first

**Pull request merged**